lib/screens/qpi/add_qpi.dart

```diff
@@ -1,29 +1,353 @@
+  bool semesterSelected = false;
+  bool courseSelected = false;
+  String title = 'Year';
+
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: AppBar(),
-      body: Text('add/edit qpi'),
+      appBar: AppBar(
+        title: Text('ADD QPI'),
+        elevation: 0,
+      ),
+      body: Center(
+        child: Container(
+          color: AppColors.PRIMARY_MAIN,
+          padding: EdgeInsets.fromLTRB(16.0, 48.0, 16.0, 96.0),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.center,
+            children: [
+              Row(
+                children: [
+                  Text(
+                    'Add $title QPI',
+                    style: GoogleFonts.dmSans(
+                      fontSize: 32.0,
+                      fontWeight: FontWeight.bold,
+                      height: 1.0,
+                      color: Colors.white,
+                    ),
+                  )
+                ],
+              ),
+              ButtonRow('Year', 'Semester', 'Course', () {
+                // Item 1 Clicked
+                setState(() {
+                  yearSelected = true;
+                  semesterSelected = false;
+                  courseSelected = false;
+
+                  title = 'Year';
+                });
+              }, () {
+                // Item 2 Clicked
+                setState(() {
+                  yearSelected = false;
+                  semesterSelected = true;
+                  courseSelected = false;
+
+                  title = 'Semester';
+                });
+              }, () {
+                // Item 3 Clicked
+                setState(() {
+                  yearSelected = false;
+                  semesterSelected = false;
+                  courseSelected = true;
+
+                  title = 'Course';
+                });
+              }),
+              SizedBox(
+                height: 24,
+              ),
+              yearSelected ? YearAddQPI() : SizedBox(),
+              semesterSelected ? SemesterAddQPI() : SizedBox(),
+              courseSelected ? CourseAddQPI() : SizedBox(),
+              Spacer(),
+              LongButton('Delete', Colors.orange, Colors.white, () {
+                Navigator.pop(context);
+              })
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class YearAddQPI extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        children: [
+          Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Text(
+                'Year Level',
+                style: GoogleFonts.dmSans(
+                  fontSize: 16.0,
+                  color: Colors.white,
+                ),
+              ),
+              SizedBox(
+                height: 8,
+              ),
+              Container(
```

Comment:
from what i've seen, the `InputField` class (and the inner text field) automatically expands to fit the parent's maximum width. we can test this sometime but we can ignore this first

- for screen size issues